website/docs/r/pi_instance.html.markdown

```diff
@@ -108,6 +108,7 @@ Review the argument references that you can specify for your resource.
 - `pi_replication_policy` - (Optional, String) The replication policy that you want to use, either `affinity`, `anti-affinity` or `none`. If this parameter is not set, `none` is used by default.
 - `pi_replication_scheme` - (Optional, String) The replication scheme that you want to set, either `prefix` or `suffix`.
 - `pi_replication_sites` - (Optional, List) Indicates the replication sites of the boot volume.
+- `pi_retain_virtual_serial_number` - (Optional, Boolean) Indicates whether attached virtual serial number will be reserved when serial assigned to instance is changed or removed.
```

Comment:
This only applies when we are deleting, we can update the description to say if the serial # will be reserver when the vm get deleted.

Reply:
What if the VSN is detached or updated then? Should it default to false? Detaching VSN gives the option to retain too.

Reply:
Yeah, I forgot about that part --  your description stands.